includes/templates/admin-auto-issuance-form.php

```diff
@@ -69,14 +69,15 @@ class="button accredible-button-outline-natural accredible-button-large"><?php e
 				<?php esc_html_e( 'Credential groups need to have been created before configuring auto issuance. If none appear, check your API key to make sure your integration is set up properly.' ); ?>
 			</div>
 
-			<form id="issuance-form" action="admin.php?page=accredible_learndash_admin_action&action=<?php echo esc_attr( $accredible_learndash_form_action ); ?>" method="post">
+			<form id="issuance-form">
 				<?php if ( 'add_auto_issuance' === $accredible_learndash_form_action ) { ?>
 					<?php wp_nonce_field( $accredible_learndash_form_action, '_mynonce' ); ?>
 				<?php } else { ?>
 					<?php wp_nonce_field( $accredible_learndash_form_action . $accredible_learndash_issuance->id, '_mynonce' ); ?>
 					<input type="hidden" name="id" value="<?php echo esc_attr( $accredible_learndash_issuance_id ); ?>">
 				<?php } ?>
 
+				<input type="hidden" name="call_action" value="<?php echo esc_attr( $accredible_learndash_form_action ); ?>">
```

Comment:
Question - Should we change the value to `edit_auto_issuance` after successfully creating an auto issuance?

Reply:
Updated.